### PR TITLE
FEAT: Added notes page

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added mockbee
- Added notes page
- added filter section(without functionality)

Files to check : 
- pages / Notes-Page / Filter
- pages / Notes-Page / Note
- pages / Notes-Page / NotesPage.jsx

### CODE REVIEW

File: src/pages/Notes-Page/Filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+.para-xsmall {
+  font-size: 1.6rem;
+  font-weight: 500;
+}
+
+/* filter-item-title */
+.old-date {
+  background-color: #fbf8cc;
+}
```
Comment (srishtigautam21) [inline src/pages/Notes-Page/Filter/Filter.css]: Use var for colors

File: src/pages/Notes-Page/Filter/Filter.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.prio-medium {
+  background-color: #a0c4ff;
+}
+.prio-high {
+  background-color: #ffadad;
+}
+
+.filter-radio {
+  width: 14px;
+  height: 14px;
```
Comment (srishtigautam21) [inline src/pages/Notes-Page/Filter/Filter.css:80]: you can use rem here

**Review by srishtigautam21 — APPROVED** [review]
Suggested some small changes
UI suggestion:
Horizontal scroll bar is there in your app which should not be there please look into it
Approving for merge

**Review by archit7448 — APPROVED** [review]
code looks proper good to merge , you can use var

**Comment by zoyatanweer** [comment]
Thankyou for your suggested changes, I have implemented it.

**Pull request merged**